test/e2e/predictor/test_raw_deployment.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from kubernetes import client
+from kfserving import (
+    constants,
+    KFServingClient,
+    V1beta1InferenceService,
+    V1beta1InferenceServiceSpec,
+    V1beta1PredictorSpec,
+    V1beta1TorchServeSpec,
+)
+from kubernetes.client import V1ResourceRequirements
+
+from ..common.utils import KFSERVING_TEST_NAMESPACE
+
+api_version = f"{constants.KFSERVING_GROUP}/{constants.KFSERVING_VERSION}"
+api_v1beta1_version = (
+    f"{constants.KFSERVING_GROUP}/{constants.KFSERVING_V1BETA1_VERSION}"
+)
+KFServing = KFServingClient(
+    config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
+
+
+def test_raw_deployment_kfserving():
+    service_name = "rawtest"
+    predictor = V1beta1PredictorSpec(
+        min_replicas=1,
+        pytorch=V1beta1TorchServeSpec(
+            storage_uri="gs://kfserving-examples/models/torchserve/image_classifier",
+            protocol_version="v1",
+            resources=V1ResourceRequirements(
+                requests={"cpu": "1", "memory": "4Gi"},
+                limits={"cpu": "1", "memory": "4Gi"},
+            ),
+        ),
+    )
+
+    annotations = dict()
+    annotations['serving.kubeflow.org/raw'] = 'true'
+
+    isvc = V1beta1InferenceService(
+        api_version=api_v1beta1_version,
+        kind=constants.KFSERVING_KIND,
+        metadata=client.V1ObjectMeta(
+            name=service_name, namespace=KFSERVING_TEST_NAMESPACE, annotations=annotations,
+        ),
+        spec=V1beta1InferenceServiceSpec(predictor=predictor),
+    )
+
+    KFServing.create(isvc)
```

Comment:
Can you add the test to send a request and assert the expected response?

Reply:
Is there a ingress in the test env?